train.py

```diff
@@ -69,14 +75,33 @@ def main(args):
     Multi-frame query:
     ```shell
     python3 train.py \
-      config/regression/mapfree/multiframe/3d3d_multi.yaml config/mapfree.yaml
+      config/regression/mapfree/multiframe/3d3d_multi.yaml \
+      config/mapfree.yaml \
+      config/mapfree_multi.yaml
     ```
     """
     parser = argparse.ArgumentParser()
-    parser.add_argument('config', help='path to config file')
-    parser.add_argument('dataset_config', help='path to dataset config file')
+    parser.add_argument('config', action='append', help='path to config file')
+    parser.add_argument('dataset_config', nargs='+', action='store',
+                        help='path to dataset config file')
+    parser.add_argument('--config', action='append', dest='config',
```

Comment:
What kind on data do you expect we need from a second config file?

Reply:
anything people want to add to their own models.